app.js

```diff
@@ -39,11 +40,11 @@ app.use((error, req, res, next) => {
 
 //psqlError
 app.use((error, req, res, next) => {
-    if (error.code === "22P02" || error.code === "23502") {
-      res.status(400).send({ msg: error.msg || "Bad Request" });
-    } else if (error.code === "23503") {
-      res.status(404).send({ msg: error.msg || "User input not found" });
-    } else next(error);
+  if (error.code === "22P02" || error.code === "23502") {
```

Comment:
You could be more specific about what each of these error codes mean so that the error message you send back is more helpful